source/jormungandr/jormungandr/scenarios/ridesharing/instant_system.py

```diff
@@ -121,11 +132,15 @@ def _make_response(self, raw_json):
 
                 user = ridesharing_ad['user']
 
-                res.driver = rsj.Individual(name=user.get('alias'),
-                                            gender=user.get('gender'),
-                                            image_url=user.get('imageUrl'),
-                                            rate=user.get('rate'),
-                                            rate_count=user.get('count'))
+                gender = user.get('gender')
+                gender_map = {'MALE': rsj.Gender.MALE,
+                              'FEMALE': rsj.Gender.FEMALE}
+
+                res.driver = rsj.Individual(alias=user.get('alias'),
+                                            gender=gender_map.get(gender, rsj.Gender.UNKNOWN),
```

Comment:
I guess None would do just the job, no?

Reply:
I hesitated.. i think the code will be more semantic given that enum...

Reply:
Yes...